nix/status-go/default.nix

```diff
@@ -22,7 +22,7 @@ let
   mobileConfigs = {
     android = {
       name = "android";
-      outputFileName = "status-go-${version}.aar";
+      outputFileName = "status-go-${strings.substring 0 7 rev}.aar";
```

Comment:
We're already calculating a shortened SHA1 above in https://github.com/status-im/status-react/blob/develop/nix/status-go/default.nix#L20, maybe we should create a variable and share the value?

Reply:
Decided to save you time and force-push that change.